.github/workflows/security.yml

```
@@ -429,3 +475,89 @@ jobs:
+        # --ignore-scripts prevents lifecycle script execution (SonarCloud S6724).
+        run: |
+          npx --yes --ignore-scripts cdxgen -t pnpm -o sbom/npm/npm.cdx.json
+      - name: Merge SBOMs
+        # 合并 Rust 和 NPM 的 SBOM 为统一文件
+        run: |
+          npx --yes --ignore-scripts @cyclonedx/cyclonedx-cli merge \
+            --input-files sbom/rust/*.cdx.json sbom/npm/*.cdx.json \
+            --output-file sbom/bom.cdx.json
+          echo "=== Merged SBOM ==="
+          python3 - <<'PY'
+import json
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)